### ISSUE TITLE
Changing `currenttime` in video gives strange results

### ISSUE THREAD

**Issue by ir-fuel** [electron/electron#7323]
- Electron version: 1.4.1
- Operating system: Mac OSX 10.11.6

I am rendering a HTML5 `video` tag in Electron. That video tag plays a 1.2GB video file that is stored on the local file system (not packaged, simply somewhere on the user's computer). Simply playing it works fine. However as soon as I change the `currenttime` value on that video it starts seeking for a very long time (even if I only add 5 seconds to the current time). Once the seek is done (I can see this by monitoring the events on the `video` element) the actual current time is still the same as before, and the video stops playing images, it only plays audio.

Some further testing showed me that it's not only by changing the `currenttime` value that this happens, but the same happens when enabling the default video controls and moving the current position around

.

Next I tested exactly the same code in Chrome 53.0.2785.116, and there everything works correctly.

With a smaller file all seems to work fine.

The html I used to test this both in Electron and Chrome, can be found at the bottom of this post. If you want to test it you'll need to replace the file link to the video file. The file I used for these tests  is located at:

https://dl.dropboxusercontent.com/u/47545939/SCHD0088.MOV

Beware when downloading, as it is 1.2GB.

here is the `index.html` file

```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
</head>

<body>

<div class="application">
  <div class="video">
    <video class="video-js vjs-default-skin" preload="auto" width="640" controls id="video"
           src="/Users/Joris/Desktop/SCHD0088.MOV"></video>
  </div>
</div>

</body>
```

and the `main.js` electron file

```
const {app, BrowserWindow} = require('electron')

let win

function createWindow () {
  win = new BrowserWindow({width: 1280, height: 800})

  win.loadURL(`file://${__dirname}/client/index.html`)

  win.webContents.openDevTools()

  win.on('closed', () => {
    win = null
  })
}

app.on('ready', createWindow)

app.on('window-all-closed', () => {
    app.quit()
})

app.on('activate', () => {
  if (win === null) {
    createWindow()
  }
})
```

**Comment by ir-fuel** [comment]
Some more info. I tested the following steps:
Open the page.
Start the video and immediately pause it.
Clear the network tab in the web inspector.
Go directly to about halfway through the video by clicking on the slider.
Check the network tab.

This is what it looks like in Chrome:
![screen shot 2016-09-23 at 15 53 54](https://cloud.githubusercontent.com/assets/1573715/18788206/4d7a3458-81a6-11e6-90f9-a7c9a8afdf5d.png)
![screen shot 2016-09-23 at 15 54 00](https://cloud.githubusercontent.com/assets/1573715/18788207/4d830682-81a6-11e6-9303-4f02684bb065.png)

This is what it looks like in Electron:
![screen shot 2016-09-23 at 15 54 38](https://cloud.githubusercontent.com/assets/1573715/18788222/57d3a6e6-81a6-11e6-97fa-ba23dbd8c0a9.png)
![screen shot 2016-09-23 at 15 54 50](https://cloud.githubusercontent.com/assets/1573715/18788223/57d3f48e-81a6-11e6-8382-a61d3bc6601e.png)

**Comment by ir-fuel** [comment]
And even more info. I had a quick look at one of your competitors, nwjs, where I tested the same thing.

I had to replace their ffmpeg lib with one that supports h264 playback, but I managed to reproduce it there too. The result is exactly the same as in Electron, however I managed to grab some logs from their console. I don't know how helpful they are. When moving the current video position I get this:

```
[16555:33547:0923/164111:ERROR:vt_video_decode_accelerator_mac.cc(649)] Illegal attempt to decode without IDR. Discarding decode requests until next IDR.
[16556:35863:0923/164116:ERROR:ffmpeg_demuxer.cc(1510)] OnReadFrameDone result=-541478725 IsMaxMemoryUsageReached=0
[16555:33919:0923/164131:ERROR:vt_video_decode_accelerator_mac.cc(649)] Illegal attempt to decode without IDR. Discarding decode requests until next IDR.
[16595:33823:0923/164135:ERROR:ffmpeg_demuxer.cc(1510)] OnReadFrameDone result=-541478725 IsMaxMemoryUsageReached=0
```

The behaviour in the network activity view is identical to the one I saw in Electron.

**Comment by ir-fuel** [comment]
More updates!

I just tested it on my windows machine and video playback works fine there.

**Comment by deepak1556** [comment]
Unable to reproduce the problem on linux with the above example, tried both seeking with the controls and `currentTime` property. `net-log` trace also seems fine. @ir-fuel what happens if you serve the file over `http` scheme ?

**Comment by ir-fuel** [comment]
Yeah I think it's mac only. As I said on Windows 10 it works fine too.
I think I tried streaming it (it's just a matter of using the Dropbox link I posted to test) but I can't recall if it worked or not.

**Comment by zcbenz** [comment]
Sorry but for an issue with a certain video file, it takes great efforts to debug while few people benefit from it, I think none of us would have time to look into this.

**Comment by ir-fuel** [comment]
I completely forgot about this issue, but it seems resolved in Electron 1.6. I presume it got fixed with an update to the Chrome renderer used.
